package.json

```diff
@@ -41,16 +41,16 @@
   "dependencies": {
     "brout": "^1.3.0",
     "listen": "^1.0.0",
-    "mocha": "^8.4.0",
+    "mocha": "^9.1.0",
```

Comment:
Are you planning to have this land in Mochify as well in the long run? Even if the only [relevant breaking change](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#900--2021-06-07) seems to be dropping Node 10 I'm wonderig if this would mean we'd have to do major version bumps on both mocaccino and Mochify?

Reply:
I was under the impression that we're stuck with mocha 8 because mocha 9 cannot be browserified (whole reason for the rewrite that I'm not getting over the finish line 🤭). Did that change?

Reply:
My memories might be off here, but from what I can recall that move from Browserify to Rollup in Mocha (which you might be referring to?) happened in some version before 8 already and was covered here https://github.com/mantoni/mocaccino.js/pull/35#discussion_r633248575 and here https://github.com/mantoni/mochify.js/issues/213#issuecomment-841668325 so I think Mocha 9 should work. That doesn't necessarily make it true though...

Reply:
This PR seems to build find when brought into mochify.js: https://github.com/mantoni/mochify.js/pull/276

@mantoni Are you referring to being stuck on an old version of browserify instead of mocha (https://github.com/mantoni/mochify.js/issues/224)?

Reply:
It used to break, I've sent a PR to fix it, but nobody replied. https://github.com/mochajs/mocha/pull/4121

This was the whole reason I went off to write a new mochify, that isn't depending on a bundler (plus some other improvements in the design). See https://github.com/mantoni/mochify.js/milestone/1

Reply:
Your PR says

> I'm maintaining mochify and would like to move away from depending on the pre-bundled Mocha.

which also aligns with what I remember: it's just not possible to browserify `browser-entry.js` file anymore because of that missing transform. Mocaccino however currently loads the pre-bundled `mocha/mocha.js` here: https://github.com/mantoni/mocaccino.js/blob/aee84ab50c0404b3346934f78dff10081a80309c/lib/mocaccino.js#L65-L67

so I think Mocha 9 should work fine as long as we consume the bundled version.

---

That being said, if I can help with finishing that rewrite in some way, I am happy to help, it will be worth it nonetheless :)

Reply:
So should we update to mocha 9 and leave a bundled mocha.js or should we stay at mocha 8 for consistency until the rewrite finishes?

Reply:
I trust you with this decision. If you have time to do the work, I'll support the upgrade.

Reply:
I removed the mocha update from this PR.  It can be added in a later PR.